### DISCUSSION TITLE
关于批量翻译PDF 的问题

### DISCUSSION THREAD

**Discussion by kylinqin** [binary-husky/gpt_academic#481] [Q&A]
导入软件依赖失败。使用该模块需要额外依赖，安装方法pip install --upgrade pymupdf tiktoken。

但pip list 发现已经安装了两个包，这是为什么？
tiktoken                 0.3.3
PyMuPDF                  1.21.1

**Comment by Yin178** [comment]
另外一个PDF功能也是这样 pip install后还是说没环境
